pkg/codegen/internal/test/testdata/resource-options.pp.go

```diff
@@ -9,7 +9,7 @@ import (
 func main() {
 	pulumi.Run(func(ctx *pulumi.Context) error {
 		provider, err := providers.Newaws(ctx, "provider", &providers.awsArgs{
-			Region: pulumi.String("us-west-2"),
+			Region: "us-west-2",
 		})
```

Comment:
These lines seem suspect, but appears to be an existing issue (i.e. does `providers`, `Newaws` exist?And  I wouldn't expect `awsArgs` to be accessible).

Though, is the `pulumi.String` -> `string` intentional?

Reply:
Yeah, this is definitely suspect. Another thing to follow up on w.r.t. Go codegen.